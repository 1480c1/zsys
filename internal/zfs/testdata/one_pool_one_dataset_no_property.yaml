pools:
  - name: rpool
    datasets:
      - name: .
