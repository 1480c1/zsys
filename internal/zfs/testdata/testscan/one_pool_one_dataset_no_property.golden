[
   {
      "Name": "testscan_one_pool_one_dataset_no_property-rpool",
      "IsSnapshot": false,
      "Mountpoint": "/",
      "CanMount": "off",
      "BootFS": "",
      "LastUsed": 0,
      "SystemDataset": ""
   }
]